recipes/nds2-client/meta.yaml

```diff
@@ -0,0 +1,116 @@
+{% set name = "nds2-client" %}
+{% set version = "0.15.3" %}
+{% set sha256 = "b39616306cb0e5f80e052bedc34d2e93ac027d0f1c8ae3d2c813d9b60c5b3e81" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or not py2k]
+
+requirements:
+  build:
+    - make
+    - cmake >=3.6
+    - swig >=3.0.7
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - libcurl
+    - krb5
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - nds2
+  commands:
+    - nds-client-config --includedir --libdir
+    - nds_query --version
+
+outputs:
+  - name: nds2-client
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - libcurl
+        - krb5
+      run:
+        - libcurl
+        - krb5
+    files:
+      - bin/nds*
+      - etc/nds2-client/
+      - include/nds2-client
+      - share/man/man*/nds*
+      - lib/libnds*
+      - lib/pkgconfig/nds2-client.pc
+      - share/man/man*/nds*
+      - share/man/man*/NDS*
+    test:
+      commands:
+        - nds-client-config --includedir --libdir
+        - nds_query --version
+    about:
+      home: https://wiki.ligo.org/DASWG/NDSClient
+      dev_url: https://git.ligo.org/nds/nds2-client
+      doc_url: https://nds.docs.ligo.org/nds2-client/Beta/User/html/
+      license: GPLv3
+      license_family: GPL
+      license_file: COPYING
+      summary: NDS2 Client interface
+      description: |
+        The NDS2 client interface allows the user to down-load LIGO data
+        from V1 and V2 LIGO Network Data Servers.
+  - name: python-nds2-client
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - {{ pin_subpackage('nds2-client', exact=True) }}
+        - libcurl
+        - krb5
+        - python
+        - numpy
+      run:
+        - {{ pin_subpackage('nds2-client', exact=True) }}
+        - libcurl
+        - krb5
+        - python
+        - numpy
+    files:
+      - lib/python*/site-packages/nds2.py*
+      - lib/python*/site-packages/__pycache__/nds2.*.pyc  # [not py27]
+      - lib/python*/site-packages/_nds2.so
```

Comment:
Please, change this part to 
```
- lib/python*/site-packages/_nds2${SHLIB_EXT}
```

Reply:
@marcelotrevisani, this doesn't work, since swig always produces a `.so` library.